packages/dev/docs/src/docs.css

```diff
@@ -51,6 +51,64 @@
   font-weight: 900;
 }
 
+/** pulled from web project id: 'uei1lip' */
```

Comment:
Allow docs to load the correct font if someone loads the page from an Arabic, Hebrew, Japanese, Korean, or Chinese locale and a component shows a i18n string.

Reply:
i think we could do what we do here in our storybook